tests/e2e/auth/suite.go

```diff
@@ -107,7 +107,7 @@ func (s *E2ETestSuite) TestCLISignGenOnly() {
 		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
 		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastSync),
 	}
-	generatedStd, err := clitestutil.ExecTestCLICmd(val.ClientCtx, bank.NewSendTxCmd(addresscodec.NewBech32Codec("cosmos")), args)
+	generatedStd, err := clitestutil.ExecTestCLICmd(val.ClientCtx, bank.NewSendTxCmd(addresscodec.NewHexCodec("cosmos")), args)
```

Comment:
Does not having `cosmos` prefix make any difference ?
```suggestion
	generatedStd, err := clitestutil.ExecTestCLICmd(val.ClientCtx, bank.NewSendTxCmd(addresscodec.NewHexCodec("")), args)
```

Reply:
Good one.
Missed this as `cli_test.go` is ignored by the `build` took because of the custom flag, hence the sdk was building correctly despite this.
Addressed in my `gov` PR 🙌

Reply:
Pushed 🙌